assets/js/post_office_search.tsx

```diff
@@ -13,25 +13,29 @@ const root = createRoot(elem);
 const { locationsSearchUrl } = elem.dataset;
 
 root.render(
-  <FullAddressSearch
-    disabled={false}
-    handleLocationSelect={null}
-    locationsURL={locationsSearchUrl}
-    noInPersonLocationsDisplay={NoInPersonLocationsDisplay}
-    onFoundLocations={() => {}}
-    registerField={() => {}}
-    resultsHeaderComponent={() => (
-      <Alert type="info" className="margin-bottom-4">
-        <strong>
-          {t('in_person_proofing.body.location.po_search.you_must_start.message', {
-            app_name: 'Login.gov',
-          })}
-        </strong>{' '}
-        <a href={t('in_person_proofing.body.location.po_search.you_must_start.link')}>
-          {t('in_person_proofing.body.location.po_search.you_must_start.link_text')}
-        </a>
-      </Alert>
-    )}
-    usStatesTerritories={UsStatesTerritories}
-  />,
+  <form>
+    <FullAddressSearch
+      noValidate
+      disabled={false}
+      handleLocationSelect={null}
+      locationsURL={locationsSearchUrl}
+      noInPersonLocationsDisplay={NoInPersonLocationsDisplay}
+      onFoundLocations={() => {}}
+      registerField={() => {}}
+      resultsHeaderComponent={() => (
+        <Alert type="info" className="margin-bottom-4">
+          <strong>
+            {t('in_person_proofing.body.location.po_search.you_must_start.message', {
+              app_name: 'Login.gov',
+            })}
+          </strong>{' '}
+          <a href={t('in_person_proofing.body.location.po_search.you_must_start.link')}>
+            {t('in_person_proofing.body.location.po_search.you_must_start.link_text')}
+          </a>
+        </Alert>
+      )}
+      usStatesTerritories={UsStatesTerritories}
+    />
+    ,
```

Comment:
Is this comma expected to be here, or leftover from previous implementation? It's causing the comma to be output visibly to the page:

![image](https://github.com/18F/identity-site/assets/1779930/4aba160d-c462-4b1b-b8ba-6d9fdd6a1134)

```suggestion
```